new_platforms/vsextension/ProjectWizard/ImportEnclaveCommand.cs

```diff
@@ -393,11 +393,20 @@ private async void Execute(object sender, EventArgs e)
                     {
                         var config3 = config as VCConfiguration3;
                         string name = config.Name;
+
+                        var clRule = config.Rules.Item("CL") as IVCRulePropertyStorage;
+                        string preprocessorDefinitions = clRule.GetUnevaluatedPropertyValue("PreprocessorDefinitions");
+                        if (isWindows)
+                        {
+                            // Allow calling getenv() in the generated host code.
+                            preprocessorDefinitions = "_CRT_SECURE_NO_WARNINGS;" + preprocessorDefinitions;
```

Comment:
Instead of defining `_CRT_SECURE_NO_WARNINGS` for the entire host, can you surround the call to `getenv` as:

```
#ifdef _MSC_VER
#pragma warning(push)
#pragma warning(disable:4996)
#endif

const char* simulation = getenv("OE_SIMULATION");

#ifdef _MSC_VER
#pragma warning(pop)
#endif
```

Reply:
Will do

Reply:
Done